crates/holochain_p2p/src/lib.rs

```diff
@@ -132,7 +132,7 @@ pub trait HolochainP2pDnaT: Send + Sync {
         options: actor::GetActivityOptions,
     ) -> actor::HolochainP2pResult<Vec<AgentActivityResponse<ActionHash>>>;
 
-    /// Get agent deterministic activity from the DHT.
+    /// Get agent activity deterministically from the DHT.
```

Comment:
This is a really important distinction. Is the must_get only returning deterministic information or is it returning everything in a deterministic way? I'd hope it's both, but do we genuinely return everything from the chain? I don't actually know.

Reply:
The must_gets that are available from the integrity are deterministic. Any agent that executes them "must" receive the same response. I don't understand what "deterministic information" would be here.